neurons/miner.py

```diff
@@ -70,6 +99,46 @@ async def forward(
             otika.protocol.SearchSynapse: The synapse object with the 'results' field set to list of the 'Document'.
         """
         bt.logging.info("received request...", query)
+
+        if str2bool(os.environ.get("MINER_SEARCH_WITH_CRAWLING", "")):
```

Comment:
Maybe let's enable this by default? We want miners with minimal experience able to get things running simply by following the instructions